### PR TITLE
 Correct CARGO URL for EVE delivery VAS

### PR DESCRIPTION
**Pull request by pankajagrawal16**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/gRpnIeCk](https://trello.com/c/gRpnIeCk)

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Jekyll can serve the site locally and auto-watch for changes:
 
     brew install node
     bundle install
-    bundle exec rake serve
-
+    bundle exec rake serve or bundle exec rake serve[env(test, qa, production)]
```
Comment (robinsk) [inline README.md]: It looks like `rake serve[env(test, qa, production)` is the command to run, or maybe `rake serve[env(qa)]`?

I guess it isn't. Maybe it reads better if we write

bundle exec rake serve

# or if you want to run for a specific environment:
bundle exec rake serve[<env>] # test, qa, or production
```

File: Rakefile [hunk 2]
```diff
@@ -18,15 +18,17 @@ task :install_dependencies do
...
+  puts "= build using jekyll with env #{args.environment}"
+  system("JEKYLL_ENV=#{args.environment} bundle exec jekyll build --trace")
   system('git rev-parse HEAD >> _site/.gitcommit')
 end
 
 desc 'Run locally using jekyll'
-task :serve => [:clean, :install_dependencies] do
-  puts '= serve using jekyll'
+task :serve, [:environment] => [:clean, :install_dependencies] do |t, args|
+  args.with_defaults(:environment => "production")
```
Comment (robinsk) [inline Rakefile]: The default should be `development` or `test` when running locally, so we hide away analytics and disqus and stuff.

**Comment by pankajagrawal16** [comment]
@robinsk Looks better ?

**Comment by pankajagrawal16** [comment]
Merging this one now @robinsk .. let me know if you have some suggestions later :p

**Pull request merged**